scripts/tt_boot_fs.py

```diff
@@ -728,11 +728,14 @@ def cksum(data: bytes):
     return calculated_checksum
 
 
-def mkfs(path: Path, env={"$ROOT": str(ROOT)}) -> bytes:
+def mkfs(path: Path, env={"$ROOT": str(ROOT)}, hex=False) -> bytes:
```

Comment:
Would be good to keep units for consistence. I think this might work? 🤷🏼‍♂️

```suggestion
def mkfs(path: Path, env={"$ROOT": str(ROOT)}, hex: Bool = False) -> bytes:
```